cameronRomo/romcom
# Feature/saving and viewing covers

What’s this PR do?
* Fixed broken img
* Add saveUserCover() to save instance of Cover class to Data Model 
  * No duplicates allowed
* Add displaySavedCover() to insert adjacent HTML to the DOM using the DM(saved covers array) to incorporate DM elements to saved covers view.

Where should the reviewer start?
*  Notice how the image embrace now loads on the deployed site
* Look in the saveUserCover() to note how data is being saved to the DM
* Look in the displaySavedCover() to note how the data from DM is updating the DOM with all its saved elements

How should this be manually tested?
* Deploy site and save randomly generated or custom made Covers, then 'click' View Saved Covers and notice how each saved cover is displayed.

Any background context you want to provide?
* Heavy on understanding the data model, DOM tree, and prototype methods as well as data types.
* Interpolation
* For loops and array indexes

What are the relevant tickets?
* N/A

Screenshots (if appropriate)
![Screen Shot 2020-08-30 at 12 49 03 PM](https://user-images.githubusercontent.com/63012953/91667087-3958de80-eabf-11ea-89d8-5abf4e6c87ea.png)
